webui/react/src/pages/Clusters/ClustersQueuedChart.tsx

```diff
@@ -45,8 +45,11 @@ const ClustersQueuedChart: React.FC<Props> = ({ poolStats }: Props) => {
         title="Avg Queue Time">
         <ClusterHistoricalUsageChart
           chartKey={viewDays}
+          formatValues={(_: uPlot, splits: number[]) =>
+            splits.map((n) => durationInEnglish(n * 1000))
+          }
           hoursByLabel={queuedStats.hoursAverage}
-          label="Queued Minutes"
+          label=" "
```

Comment:
**nit question:** does an empty string of `""` not work where you have to have that one space character?

Reply:
I think so, label being empty defaults to "GPU hours"